float64 ph
float64 conductivity
float64 temperature
float64 dissolved_oxygen
float64 redox_potential
string date
bool is_connected
